pulsar-broker/src/test/java/org/apache/pulsar/broker/zookeeper/ClusterMetadataSetupTest.java

```diff
@@ -86,6 +88,32 @@ public void testReSetupClusterMetadata() throws Exception {
         assertEquals(data1, data3);
     }
 
+    public void testSetBundleNumberForDefaultNamespace() throws Exception {
+
+        String[] args = {
+                "--cluster", "testReSetupClusterMetadata-cluster",
+                "--metadata-store", "zk:127.0.0.1:" + localZkS.getZookeeperPort() + "/metastore",
+                "--configuration-store", "127.0.0.1:" + localZkS.getZookeeperPort() + "/configurationStore",
+                "--web-service-url", "http://127.0.0.1:8080",
+                "--web-service-url-tls", "https://127.0.0.1:8443",
+                "--broker-service-url", "pulsar://127.0.0.1:6650",
+                "--broker-service-url-tls","pulsar+ssl://127.0.0.1:6651",
+                "--bundle-numbers", "64",
```

Comment:
This could be confusing because if the namespace is already existing, it won't have any effect. Also it's not clear in the command line that this will be related to the `public/default` namespace.

Another approach could be to remove the 16 bundles in the hardcoded from and instead just use the default number of bundles which is set in `broker.conf`

Reply:
> Another approach could be to remove the 16 bundles in the hardcoded from and instead just use the default number of bundles which is set in `broker.conf`
This is a better way.